PowerUp/app/src/main/java/powerup/systers/com/data/entities/Question.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * @desc this class holds getter and setter methods for each question and the scenario it belongs to
+ * examples include getQuestionDescription() and setScenarioID()
+ */
+
+package powerup.systers.com.data.entities;
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.PrimaryKey;
+
+@Entity(tableName = "Question")
+public class Question {
+    @PrimaryKey
+    private Integer questionID;
+
+    private Integer scenarioID;
+
+    private String questionDescription;
+
+
+
+    public Integer getQuestionID() {
+        return questionID;
+    }
+
+    public void setQuestionID(Integer qId) {
+        this.questionID = qId;
+    }
+
+    public String getQuestionDescription() {
+        return questionDescription;
+    }
+
+    public void setQuestionDescription(String qDes) {
+        this.questionDescription = qDes;
+    }
+
+    public Integer getScenarioID() {
+        return scenarioID;
+    }
+
+    public void setScenarioID(Integer scenarioID) {
+        this.scenarioID = scenarioID;
+    }
+
+
```

Comment:
blank line can be removed

Reply:
Yeah, Only one line is needed as we'll have Constructor+getter/setter after it. I'll update it.